cmd/kpod/logs.go

```diff
@@ -4,8 +4,9 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/projectatomic/libpod/libpod"
+
```

Comment:
Remove the extra spaces around the github lines.

Reply:
fixed